backend/src/Altinn.Authorizationadmin/Altinn.AuthorizationAdmin.Core/Altinn.AuthorizationAdmin.Core.csproj

```diff
@@ -10,6 +10,7 @@
     <PackageReference Include="Azure.Storage.Blobs" Version="12.13.1" />
     <PackageReference Include="Azure.Storage.Queues" Version="12.11.1" />
     <PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="6.0.0" />
+    <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
```

Comment:
There's been a decision to move away from dependencies to Newtonsoft.Json. Use System.Text.Json instead